demo/README.md

```diff
@@ -0,0 +1,28 @@
+# Demos
```

Comment:
This folder will not remain here once the collection is ready (all these things could be moved to some user guides). I included it just to give an example of how this collection can be used with some ready-made playbooks. However, let's keep it here for the time being and then decide where and how to move it.

Reply:
Makes sense to me!